crates/ty_python_semantic/resources/mdtest/type_properties/constraints.md

```diff
@@ -601,3 +601,40 @@ def _[T, U]() -> None:
     # revealed: ty_extensions.ConstraintSet[always]
     reveal_type(~union | union)
 ```
+
+## Other simplifications
+
+When rendering a constraint set, we transform the internal BDD representation into a DNF formula
+(i.e., the logical OR of several clauses, each of which is the logical AND of several constraints).
+This section contains several examples that show that we simplify the DNF formula as much as we can
+before rendering it.
+
+```py
+from ty_extensions import range_constraint
+
+def _[T, U]():
```

Comment:
nit: I find all these functions being called `_` a bit odd for the tests in this file specifically, because my brain keeps parsing `@_` as a single glpyh -- the underscore looks like a continuation of the `@` to me 😆 would it be possible to call the function `f` (or really, anything that doesn't start with `_`)?